generated/nirfmxinstr/nirfmxinstr.proto

```diff
@@ -1054,6 +1054,7 @@ message InitializeResponse {
   int32 status = 1;
   nidevice_grpc.Session instrument = 2;
   int32 is_new_session = 3;
+  string error_message = 4;
```

Comment:
Do we not need it for GetNIRFSASession and GetNIRFSASessionArray? I guess since those are adding a dependent session and have an input of an RFmx session they should be fine to look up the error message on failure like normal, right?

Reply:
Yeah, any errors for those should be associated with the initiating RFmx session.